src/toolbar/sticky/stickytoolbarview.js

```diff
@@ -169,3 +183,41 @@ export default class StickyToolbarView extends ToolbarView {
  * @observable
  * @member {Boolean} ui.stickyToobar.StickyToolbarViewModel#isSticky
  */
+
+/**
+ * Controls the `left` CSS style of the toolbar.
+ *
+ * @readonly
+ * @observable
+ * @member {String} ui.stickyToobar.StickyToolbarViewModel#left
```

Comment:
Why is it public together with `marginLeft`? I don't know why we need them in the first place (see one of the previous comments), but it looks like they are only an implementation detail.